apps/server/src/modules/tool/external-tool/uc/external-tool.uc.ts

```diff
@@ -122,6 +138,9 @@ export class ExternalToolUc {
 
 		const currentExternalTool: ExternalTool = await this.externalToolService.findById(toolId);
 
+		const updatedConfig = this.encryptLtiSecret(externalToolUpdate);
```

Comment:
You have to do this before line 137 where `externalToolUpdateProps` is extracted from `externalToolUpdate`

Reply:
And why not do it in one line like `externalToolUpdate.config = this.encryptLtiSecret(externalToolUpdate);`?

Reply:
good point, thanks